frontend/src/components/authchooser/index.tsx

```diff
@@ -234,16 +242,18 @@ export function PureAuthChooser({
   return (
     <ClusterDialog useCover onClose={onClose} aria-labelledby="authchooser-dialog-title">
       {testingAuth ? (
-        <Box textAlign="center">
+        <Box component="main" textAlign="center">
           <DialogTitle ref={focusedRef} id="authchooser-dialog-title">
             {testingTitle}
           </DialogTitle>
           <Loader title={t('Testing auth')} />
         </Box>
       ) : (
-        <Box display="flex" flexDirection="column" alignItems="center">
+        <Box component="main" display="flex" flexDirection="column" alignItems="center">
           <DialogTitle ref={focusedRef} id="authchooser-dialog-title">
-            {title}
+            <Typography variant="h1" className={classes.title}>
+              {title}
+            </Typography>
```

Comment:
Yeah, all the dialog views need a good heading structure (h1, h2 etc) and navigation landmarks (main). It already correctly disabled the other content on the page. So with this change we have a main, and we have a h1.

Changing the font sizing keeps it as before whilst allowing the use of the h1 tag. The design of the header tags should be separate from the use of semantically correct document structure.

This is a bug in the Material UI component (luckily it was reported long ago and is fixed in newer versions). Ultimately, we should move to newer Material UI versions to get these and other fixes... but that's a bit bigger of a change because of the major version changes. When we update to the new version we can remove all this code.